posthog/models/test/test_batch_export_model.py

```diff
@@ -0,0 +1,362 @@
+import csv
+import datetime as dt
+import io
+import time
+from uuid import uuid4
+
+from boto3 import resource
+from botocore.client import Config
+from freezegun import freeze_time
+from temporalio.client import (
+    ScheduleSpec,
+    WorkflowExecutionDescription,
+    WorkflowExecutionStatus,
+)
+from temporalio.service import RPCError
+
+from posthog.clickhouse.client import sync_execute
+from posthog.models import (
+    BatchExport,
+    BatchExportDestination,
+    BatchExportRun,
+    BatchExportSchedule,
+)
+from posthog.settings import (
```

Comment:
I think we should be using `from django.conf import settings` here right? As the canonical Django way. Or does that not really matter here?

Reply:
the only thing it matters for is testing: `override_settings` works only with `from django.conf import settings`, while in the other case, you'll have to patch the setting itself. Makes it convenient to use django.conf

Reply:
I meant that it makes no difference in this PR, but yeah, I do see the value of `override_settings` and just overall being consistent with everything else 👍 I've made the switch to use `django.conf.settings` and used `override_settings` instead of `mock` patching in the test file.